smart-contracts/helpers/bridge.js

```diff
@@ -0,0 +1,51 @@
+// see https://docs.connext.network/resources/deployments as referencce
+
+const mumbai = {
```

Comment:
ok let's move this to the networks package please!

Reply:
done https://github.com/unlock-protocol/unlock/pull/11970